Src/BootCamp.Chapter/Logging/LoggerFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter.Logging
+{
+    public static class LoggerFactory
```

Comment:
A factory usually works that given a key, it creates the object you want (IObject). This is not a typical factory pattern. but for now it's fine.